### ISSUE TITLE
Comment, Uncomment, and delete commands are not working.

### ISSUE THREAD

**Issue by hameem99** [Chakroun-Anas/turbo-console-log#203]
Currently I am facing an issue while I tried to delete. I am using Ubuntu 22.04.2. I changed the keybinding for my vscode. I tried deleting before, but now it's not working at all. I only changed my keybinding for display log command. Not for other commands.

**Comment by e1uone** [comment]
Noticed the same issue

**Comment by sb-pramod92** [comment]
Yes, I am facing the same issue. @Chakroun-Anas , please could you look into this issue?

Thanks & regards
Pramod Tolani

**Comment by GPSTEP** [comment]
I am facing the same issue

**Comment by caicai1998** [comment]
I also have this problem

**Comment by NikolaKrstic94** [comment]
It's not working for me either. I'm on Windows with latest VSCode 1.76.2

**Comment by Chakroun-Anas** [comment]
Hello guys, 

Sorry for the inconvenience, i just pushed a new version that deals with this `v2.7.2`, you should be able with now to comment, uncomment and delete all the log messages inserted by the extension using `console.log` as the debugging function, a new version is coming soon to match the fact that the debugging function can be customised to any other debugging function with the ability to make these commands works too.

Would please test the new version and give me your feedback? Thx. 

NB: The extension use the prefix and the delimiter properties to detect the log messages to comment, uncomment or delete (only console.log function is handled for now).

**Comment by sb-pramod92** [comment]
Hi @Chakroun-Anas , It's working now.

Thanks for your time.

Regards
Pramod Tolani
